src/utils/notifyIssues.js

```diff
@@ -39,7 +39,33 @@ async function getLinkedIssueNumbers(github, prNumber, repoOwner, repoName) {
   return linkedIssues.map(issue => issue.number)
 }
 
-async function notifyIssues(githubClient, owner, repo, release) {
+function createCommentBody(
+  shouldPostNpmLink,
+  packageName,
+  packageVersion,
+  releaseUrl
+) {
+  const npmUrl = `https://www.npmjs.com/package/${packageName}/v/${packageVersion}`
+
+  if (shouldPostNpmLink) {
+    return `🎉 This issue has been resolved in version ${packageVersion} 🎉 \n\n
+  The release is available on: \n * [npm package](${npmUrl}) 
+  \n * [GitHub release](${releaseUrl}) 
+  \n\n Your **[optic](https://github.com/nearform/optic-release-automation-action)** bot 📦🚀`
+  }
+
+  return `🎉 This issue has been resolved in version ${packageVersion} 🎉 \n\n
+  The release is available on: \n * [GitHub release](${releaseUrl}) 
+  \n\n Your **[optic](https://github.com/nearform/optic-release-automation-action)** bot 📦🚀`
+}
```

Comment:
I believe these strings don't need to contain \n, you can simply go to new lines. You may have to put more new lines than you expect for markdown to insert blank lines, for overall templates strings don't need to mix with explicit new lines.

Reply:
The requested changes are done.